bulma-rtl.sass

```diff
@@ -0,0 +1,10 @@
+/*! bulma.io v0.4.0 | MIT License | github.com/jgthms/bulma */
+@charset "utf-8"
+
+$right: left
+$left: right
+
+html
+  direction: rtl
+
+@import "bulma"
```

Comment:
ok. for example I can say Twitter is the most known BIDI example. you can see support both for LTR and RTL on the same page. but they did this using js by switching between dir="ltr" and dir="rtl". ok I agree with you about :dir but as you said we need to wait (maybe 1-2 years at least).

that's my pleasure that you reviewed my work and I learnt about :dir now so we can use it later :) . so for now lets wait for maintainers to decide.